Plan/common/src/main/java/com/djrapitops/plan/storage/database/SQLDB.java

```diff
@@ -58,28 +63,37 @@
  */
 public abstract class SQLDB extends AbstractDatabase {
 
+    private static final List<Repository> DRIVER_REPOSITORIES = Collections.singletonList(
+            new StandardRepository("https://papermc.io/repo/repository/maven-public/")
```

Comment:
I think maven central could be added here in case papermc ever goes down (However unlikely that is)